torch/optim/adam.py

```diff
@@ -260,6 +260,12 @@ def _single_tensor_adam(params: List[Tensor],
             grad = grad.add(param, alpha=weight_decay)
 
         # Decay the first and second moment running average coefficient
+        if torch.is_complex(param):
```

Comment:
The comment above needs to be moved down

Reply:
Agreed!